### PR TITLE
Added ripple effect

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Closing #477 

As the ripple effect is only for API level 21 and greater , I came with another method which resembles ripple effect in the lower API's.
It is just adding a attribute called
**android:background="?attr/selectableItemBackground"**
It works as a ripple effect(very little ripple effect / have to observe carefully) but not as much as it.

@mhutti1 @kelson42 @srv-twry  please take a look at it.

### CODE REVIEW

**Comment by srv-twry** [comment]
@RajaVamsi11 Please attach a GIF. Even the PR template consists of the field for it.

File: app/src/main/res/layout/help.xml [hunk 1]
```diff
@@ -57,17 +57,23 @@
...
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_margin="@dimen/dimen_3dp"
-        android:background="@drawable/get_content_btn_shape"
-        android:text="@string/menu_zim_manager"
-        android:textAllCaps="false"
-        android:textColor="@color/white"
-        android:textSize="@dimen/dimen_20sp"/>
+      <LinearLayout
+          android:layout_width="match_parent"
```
Comment (mhutti1) [inline app/src/main/res/layout/help.xml]: 2 space indentation

**Comment by RajaVamsi11** [comment]
@srv-twry  you asked me many times to present with GIF.
But my android phone API level is 19.It is not able to get instant run in android studio , as i was using emulator it is tough to get GIF.please pardon me>.

**Comment by RajaVamsi11** [comment]
@mhutti1  I have corrected the indentation problem.Take a look.

**Comment by abdulwd** [comment]
Three commits for a change in single file are too much. Try to **squash** your commits, it will make the commit history more presentable.

**Comment by RajaVamsi11** [comment]
@abdulwd  Done.

**Comment by RajaVamsi11** [comment]
@mhutti1  Take a look at this :)

**Pull request merged**